presto-hive/src/main/java/com/facebook/presto/hive/parquet/reader/ParquetColumnReader.java

```diff
@@ -237,7 +246,6 @@ private void seek()
             int offset = Math.min(remainingValueCountInPage, readOffset - valuePosition);
             skipValues(offset);
             valuePosition = valuePosition + offset;
-            updatePosition(offset);
```

Comment:
what is the reason behind this change?

Reply:
this line was moved to skipValues method